packages/config/src/bin/build-script.ts

```diff
@@ -16,19 +18,20 @@ function buildScript(argv, cwd) {
     cwd,
     shell: true,
   });
-  subprocess.on('error', function (err) {
-    logError(err);
+  subprocess.on('error', function (err: any) {
```

Comment:
Again I think ideally we'd avoid the use of `any`. Maybe we could add todo's to follow up this migration

Reply:
If we switch to an `import` statement we can use the default type. Let me do that